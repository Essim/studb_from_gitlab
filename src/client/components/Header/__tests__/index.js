import React from 'react';
import {Header} from '../';
import { shallowToJson } from 'enzyme-to-json';

const mockSignin = {user: {},
    connected: false,
};

const mockDivisions= {
        divisions: [],
};

const mockLogout = jest.fn();

describe('Header', () => {
    it('should render correctly', () => {
        const output = shallow(<Header signin={mockSignin} divisions={mockDivisions} logout={mockLogout}/>);
        expect(shallowToJson(output)).toMatchSnapshot();
    });
});
