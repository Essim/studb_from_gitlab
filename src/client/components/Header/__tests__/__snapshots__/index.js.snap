// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly 1`] = `
<header
  className="main-header"
>
  <Link
    className="logo"
    replace={false}
    to="/"
  >
    <span
      className="logo-mini"
    >
      <b>
        STU
      </b>
      DB
    </span>
    <span
      className="logo-lg"
    >
      <b>
        Student
      </b>
      DB
    </span>
  </Link>
  <nav
    className="navbar navbar-static-top"
  >
    <div
      className="navbar-custom-menu"
    >
      <ul
        className="nav navbar-nav"
      >
        <li>
          <Route
            component={[Function]}
            path="/"
          />
        </li>
      </ul>
    </div>
    <Connect(SignUp) />
  </nav>
</header>
`;
